tutorials/holohub_operators_external_applications/holohubCLI_wrapper

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Pulls the holohub CLI from the holohub repo
+if [ ! -d "${SCRIPT_PATH}/utilities/cli" ]; then
```

Comment:
How should we prompt the user to update their CLI checkout when new changes or features are available? Will this result in users potentially having different versions of the CLI locally depending on when they first ran the script?

Doesn't have to be resolved in this initial commit, but whether we can pin or otherwise version the HoloHub CLI would be something to think about.

Reply:
Added some comments in the script. We should definitely improve.